src/main/java/org/spongepowered/api/entity/player/gamemode/GameModes.java

```diff
@@ -34,6 +34,8 @@
     public static final GameMode ADVENTURE = null;
     public static final GameMode SPECTATOR = null;
 
```

Comment:
The other members represent actual gamemodes. `NOT_SET` is special, because it represents the absence of a gamemode. rather than a meaningful value. Thus, it is logically separated, and is therefore separated in code by an empty line.